great_expectations/checkpoint/toolkit.py

```diff
@@ -166,10 +166,6 @@ def get_checkpoint(
         raise ge_exceptions.CheckpointNotFoundError(
             message=f'Non-existent Checkpoint configuration named "{key.configuration_key}".\n\nDetails: {exc_ik}'
         )
-    except ValidationError as exc_ve:
-        raise ge_exceptions.InvalidCheckpointConfigError(
-            message="Invalid Checkpoint configuration", validation_error=exc_ve
-        )
```

Comment:
This feature mirrors the functionality from Checkpoint - this ValidationError has no place in this method as we are only retrieving values from the store (not running schema checks). This check is absent from the Profiler implementation.